samples/videoDecodePerf/videodecodeperf.cpp

```diff
@@ -42,6 +42,7 @@ void DecProc(RocVideoDecoder *p_dec, VideoDemuxer *demuxer, int *pn_frame, doubl
     uint8_t *p_video = nullptr;
     int64_t pts = 0;
     double total_dec_time = 0.0;
+    p_dec->SetDecoderSessionID(0);
```

Comment:
Why are you using the number 0 as the session ID here? You need to use a unique ID for each session. For example, you can use `std::this_thread::get_id()`.